frontend/apps/marketing/cicd/3-app/template.yml.erb

```
@@ -755,3 +755,9 @@ Outputs:
   SiteDomainName:
     Description: Fully qualified domain name of the marketing site
     Value: !Sub "${SubdomainName}.${BaseDomainName}"
+  CloudFrontDistributionId:
+    Value:
+      Ref: CloudFrontDistribution
+    Export:
+      Name:
+        'Fn::Sub': '${AWS::StackName}-CloudFrontDistributionId'
```

GitHub Actions is fetching the Output from this Stack and there aren't any other Stacks Importing this Exported value, so we can eliminate the Export. Also suggest we adopt the slightly more readable `!Ref` syntax as the rest of the template has been updated to use that syntax.
```suggestion
  CloudFrontDistributionId:
    Value: !Ref CloudFrontDistribution
```